pdp/static/js/pdp_controls.js

```diff
@@ -526,34 +526,30 @@ RasterDownloadLink.prototype = {
         return url;
     },
     getSingleCellDownloadLink: function (bounds, feature) {
-        if (this.ext !== "aig") {
-            var props = {
-                dl_url: this.dl_url,
-                ext: this.ext,
-                varname: this.varname,
-                trange: this.trange,
-                // This is a single cell, so left === right and top == bottom,
-                // so don't have to use both. Bonus: we don't have to account
-                // for ascending vs. descending lat, lon dimensions here.
-                xrange: `${bounds.left}:${bounds.left}`,
-                yrange: `${bounds.bottom}:${bounds.bottom}`
-            };
-
-            var url = this.url_template;
-            var matches = url.match(/\{[a-z_]+\}/g);
-            matches.forEach(
-                function (pattern, index, array) {
-                    var id = pattern.replace(/[{}]/g, '');
-                    url = url.replace(pattern, props[id]);
-                },
-                this
-            );
-            if (url = window.prompt("Would you like to download the following subset of data?", url)) {
-                location.href = url
-            };
-        } else {
-            alert("Sorry, Arc/Info ASCII output is not currently supported for this feature.\nPlease select a different data format and try again.");
-        }
+        var props = {
```

Comment:
FYI, in modern JS `var` is deprecated in favour of `const` for unvarying values or `let` for varying ones. `var` is unscoped; `const` and `let` are scoped, and there are further advantages that avoid some subtle bugs.

Reply:
But this is the Python 3 branch, so my comment is just general and probably you should not get hung up on improving the very old JS.